docs/config-file-curations-yml.md

```diff
@@ -52,6 +52,7 @@ The structure of the curations file consist of one or more `id` entries:
 - id: "package identifier."
   curations:
     comment: "An explanation why the curation is needed or the reasoning for a license conclusion"
+    purl: "[Valid](https://github.com/package-url/purl-spec) package URL."
```

Comment:
Shouldn't you better curate to something that really is a valid PURL syntax-wise?